### PR TITLE
Add Twenty Seventeen block patterns

### PR DESCRIPTION
**Pull request by kjellr**
Builds on #550.

- Fixes pattern names.
- Gets php docbloc in sync with other file headers.
- Adds textdomain to translations.
- Updates text so it uses no "lorem ipsum". 
- Fixes images.
- Adds image license information. 

Trac ticket: https://core.trac.wordpress.org/ticket/51100

---
**This Pull Request is for code review only. Please keep all other discussion in the Trac ticket. Do not merge this Pull Request. See [GitHub Pull Requests for Code Review](https://make.wordpress.org/core/handbook/contribute/git/github-pull-requests-for-code-review/) in the Core Handbook for more details.**

### CODE REVIEW

File: src/wp-content/themes/twentyseventeen/inc/block-patterns.php [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+ * Register Block Patterns.
+ */
+if ( function_exists( 'register_block_pattern' ) ) {
+	register_block_pattern(
+		'twentyseventeen/large-heading-with-button',
+		array(
+			'title'      => __( 'Large Heading with Button', 'twentyseventeen' ),
+			'categories' => array( 'twentyseventeen' ),
+			'content'    => '<!-- wp:heading {"level":1,"textColor":"black","style":{"typography":{"fontSize":50}}} -->
+            <h1 class="has-black-color has-text-color" style="font-size:50px">' . __( 'We can put together marketing campaigns and attract leads through search engine optimization.', 'twentyseventeen' ) . '</h1>
```
Comment (kjellr) [inline src/wp-content/themes/twentyseventeen/inc/block-patterns.php:31]: Do these strings need to be translated _and_ escaped? I'd been escaping them for other themes just to be safe, but wasn't sure.

Comment (ryelle) [reply src/wp-content/themes/twentyseventeen/inc/block-patterns.php:31]: This came up in 2021 https://github.com/WordPress/twentytwentyone/issues/479 and I'm not sure the final answer — for the older themes I think the standard is to not escape, [translations are considered trusted](https://core.trac.wordpress.org/ticket/30724) (so… maybe we need to remove escaping from 2020 & 2019 too).

File: src/wp-content/themes/twentyseventeen/inc/block-patterns.php [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+            <!-- wp:paragraph {"style":{"typography":{"lineHeight":"3"}}} -->
+            <p style="line-height:3"><a href="#"><strong>' . __( 'See Case Study', 'twentyseventeen' ) . ' →</strong></a></p>
+            <!-- /wp:paragraph --></div>
+            <!-- /wp:column -->
+            <!-- wp:column -->
+            <div class="wp-block-column"><!-- wp:spacer {"height":254} -->
+            <div style="height:254px" aria-hidden="true" class="wp-block-spacer"></div>
+            <!-- /wp:spacer -->
+			<!-- wp:image {"className":"size-large"} -->
+			<figure class="wp-block-image size-large"><img src="' . get_template_directory_uri() . '/assets/images/white-border.jpg" alt="' . __( 'White border', 'twentyseventeen' ) . '"/></figure>
```
Comment (carolinan) [inline src/wp-content/themes/twentyseventeen/inc/block-patterns.php:71]: This URl needs to be escaped.

File: src/wp-content/themes/twentyseventeen/inc/block-patterns.php [hunk 3]
```diff
@@ -0,0 +1,199 @@
...
+            <!-- wp:paragraph {"align":"left"} -->
+            <p class="has-text-align-left"><a href="#">' . __( 'See Case Study', 'twentyseventeen' ) . ' →</a></p>
+            <!-- /wp:paragraph --></div></div>
+            <!-- /wp:group --></div>
+            <!-- /wp:column -->
+            <!-- wp:column -->
+            <div class="wp-block-column"><!-- wp:group -->
+            <div class="wp-block-group"><div class="wp-block-group__inner-container">
+			<!-- wp:image {"align":"center","sizeSlug":"large","className":"is-style-default"} -->
+			<div class="wp-block-image is-style-default"><figure class="aligncenter size-large"><img src="' . get_template_directory_uri() . '/assets/images/direct-light.jpg" alt="' . __( 'Direct Light', 'twentyseventeen' ) . '"/></figure></div>
```
Comment (carolinan) [inline src/wp-content/themes/twentyseventeen/inc/block-patterns.php:128]: Same here, the image url needs to be escaped

File: src/wp-content/themes/twentyseventeen/inc/block-patterns.php [hunk 4]
```diff
@@ -0,0 +1,199 @@
...
+            <!-- wp:paragraph {"align":"left"} -->
+            <p class="has-text-align-left"><a href="#">' . __( 'See Case Study', 'twentyseventeen' ) . ' →</a></p>
+            <!-- /wp:paragraph --></div></div>
+            <!-- /wp:group --></div>
+            <!-- /wp:column -->
+            <!-- wp:column -->
+            <div class="wp-block-column"><!-- wp:group -->
+            <div class="wp-block-group"><div class="wp-block-group__inner-container">
+			<!-- wp:image {"align":"center","sizeSlug":"large","className":"is-style-default"} -->
+			<div class="wp-block-image is-style-default"><figure class="aligncenter size-large"><img src="' . get_template_directory_uri() . '/assets/images/white-border.jpg" alt="' . __( 'White border', 'twentyseventeen' ) . '"/></figure></div>
```
Comment (carolinan) [inline src/wp-content/themes/twentyseventeen/inc/block-patterns.php:114]: Same here, the image url needs to be escaped

File: src/wp-content/themes/twentyseventeen/inc/block-patterns.php [hunk 5]
```diff
@@ -0,0 +1,199 @@
...
+			'categories' => array( 'twentyseventeen' ),
+			'content'    => '<!-- wp:spacer -->
+            <div style="height:100px" aria-hidden="true" class="wp-block-spacer"></div>
+            <!-- /wp:spacer -->
+            <!-- wp:columns {"verticalAlignment":"top"} -->
+            <div class="wp-block-columns are-vertically-aligned-top"><!-- wp:column -->
+            <div class="wp-block-column"><!-- wp:group -->
+            <div class="wp-block-group"><div class="wp-block-group__inner-container">
+			<!-- wp:image {"align":"center","sizeSlug":"large","className":"is-style-default"} -->
+			<div class="wp-block-image is-style-default"><figure class="aligncenter size-large"><img src="' . get_template_directory_uri() . '/assets/images/stripes.jpg" alt="' . __( 'Black Stripes', 'twentyseventeen' ) . '"/></figure></div>
```
Comment (carolinan) [inline src/wp-content/themes/twentyseventeen/inc/block-patterns.php:100]: Same here, the image url needs to be escaped

File: src/wp-content/themes/twentyseventeen/inc/block-patterns.php (hunk 2 above)
Comment (kjellr) [reply src/wp-content/themes/twentyseventeen/inc/block-patterns.php:71]: @carolinan [see the discussion above](https://github.com/WordPress/wordpress-develop/pull/606#discussion_r508856895). I'm happy to escape these if they need to be escaped, but there is some confusion around it.

**Comment by kjellr** [comment]
@carolinan, those URIs should all be escaped now. Thanks!

**Comment by hellofromtonya** [comment]
Resolved in changeset https://core.trac.wordpress.org/changeset/49584

**Pull request closed without merging**